Sources/FoundationEssentials/TimeCalculations.swift

```diff
@@ -0,0 +1,292 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
```

Comment:
This doesn't seem correct

Reply:
You are right. I copied the header from another file and it turns out that a bunch of files in here have the wrong copyright header.